molecule/openstack/molecule.yml

```diff
@@ -14,6 +14,9 @@ provisioner:
   name: ansible
   lint:
     name: ansible-lint
+    options:
+      x:
+        - "204"
```

Comment:
Since this option is not descriptive by itself, it might be worth adding a comment so nobody has to go look up what '204' means.